### PR TITLE
Swerve Drive: Fix #26 and #9

### LINKED ISSUE #26: Swerve drive modules follow target even if target is 360+
If the module falls behind the target heading it keeps spinning, this is mainly due to the target increasing even if it rotates a full rotatio ahead of the actual.

@KhanSimeoni has a way to fix this, if the target leads the actual by more than 1 rotatation, use mod to remove the rotation, ie if the target is 640 degrees, and the current pose is 120 degrees, the new target will be truncated to 280

### PR DESCRIPTION
**Pull request by KhanSimeoni**

### CODE REVIEW

File: rio/subsystems/drivetrain.py [hunk 1]
```diff
@@ -290,20 +302,20 @@ def getCurrentState(self):
...
         # if self.constants["steer_id"] == 1:
         # print(encoder)
 
         # Divide encoder by ratio of encoder rotations to wheel rotations, times 2pi
-        radians = encoder * (math.pi * 2)
+        self.radians = self.encoder * (math.pi * 2)
 
         # Construct a rotation2d object
-        rot = geometry.Rotation2d(radians)
+        self.rot = geometry.Rotation2d(self.radians)
```
Comment (KenwoodFox) [inline rio/subsystems/drivetrain.py:338]: Why are these members? `self.`

File: rio/subsystems/drivetrain.py [hunk 2]
```diff
@@ -273,10 +273,22 @@ def setDesiredState(self, newState):
...
+        elif self.angleSum + (2 * math.pi) < self.radians:
+            self.angleSum = self.angleSum + (2 * math.pi)
+
         currentRef = self.angleSum / (
             2 * math.pi
         )  # The sum (radians) converted to rotations (of the steer wheel)
 
+        if self.constants["steer_id"] == 1:
+            print(self.angleSum, "Angle Sum | ", self.radians, "Radians")
+
```
Comment (KenwoodFox) [inline rio/subsystems/drivetrain.py]: This should be removed

File: rio/constants/robot_hardware.yaml [hunk 3]
```diff
@@ -54,8 +54,8 @@ drivetrain:
       # Max amount of I that can accumulate
       maxi: 1                   # Updated 2/8/22 by Joe
       # Max and min values the PID can assign to the motor
-      min_power: -0.85          # Updated 2/8/22 by Joe
-      max_power: 0.85           # Updated 2/8/22 by Joe
+      min_power: -0.85          # Updated 2/12/22 by Joe
+      max_power: 0.85           # Updated 2/12/22 by Joe
```
Comment (KenwoodFox) [inline rio/constants/robot_hardware.yaml:62]: Im in it :)

**Review by KenwoodFox — CHANGES_REQUESTED** [review]

File: rio/subsystems/drivetrain.py (hunk 1 above)
Comment (Kredcool) [reply rio/subsystems/drivetrain.py:338]: the self.s are needed otherwise the code doesn't work

Comment (KenwoodFox) [reply rio/subsystems/drivetrain.py:338]: self.radians does not need to exist outside the scope of this method.

**Pull request closed without merging**